app/src/main/kotlin/com/netguru/android/arlocalizeralternative/feature/location/domain/ILocationRepository.kt

```diff
@@ -0,0 +1,16 @@
+package com.netguru.android.arlocalizeralternative.feature.location.domain
+
+import com.google.android.gms.maps.model.LatLng
+import com.netguru.arlocalizerview.location.LocationData
+import io.reactivex.Single
+
+interface ILocationRepository {
+    fun getDestinations(
+        amenity: String, southWestLatitude: Double,
```

Comment:
move `southWestLatitude: Double` to newline to be consistent with other parameters